Advanced Collaborations/projects/PatientMonitor.json

```
@@ -0,0 +1,451 @@
+    "target" : "procedure/patient.monitoring.HeartRate.HeartRate_MonitorReading_EventHandler_LoopWhileStateUpdate"
+  }, {
+    "source" : "services/patient.monitoring.HeartRate",
+    "target" : "procedure/patient.monitoring.HeartRate.HeartRate_MonitorReading_EventHandler_LoopWhileStateGet"
+  }, {
+    "source" : "services/patient.monitoring.HeartRate",
+    "target" : "procedure/patient.monitoring.HeartRate.HeartRate_MonitorReading_EventHandler_LoopWhileStateReset"
+  }, {
+    "source" : "app/patient.monitoring.HeartRate_MonitorReading_EventHandler",
+    "target" : "procedure/patient.monitoring.HeartRate.HeartRate_MonitorReading_EventHandler_LoopWhileStateGet"
+  }, {
+    "source" : "app/patient.monitoring.HeartRate_MonitorReading_EventHandler",
+    "target" : "procedure/patient.monitoring.HeartRate.HeartRate_MonitorReading_EventHandler_LoopWhileStateUpdate"
+  }, {
+    "source" : "procedure/patient.monitoring.HeartRate.HeartRate_MonitorReading_EventHandler_LoopWhileStateReset",
+    "target" : "services/patient.monitoring.HeartRate"
+  }, {
+    "source" : "procedure/patient.monitoring.HeartRate.HeartRate_MonitorReading_EventHandler_LoopWhileStateGet",
+    "target" : "services/patient.monitoring.HeartRate"
+  }, {
+    "source" : "services/patient.monitoring.HeartRate",
+    "target" : "procedure/patient.monitoring.HeartRate.Active_HeartRate_MonitorReading_EventHandlerUpdateAssignCollaborator"
+  }, {
+    "source" : "procedure/patient.monitoring.HeartRate.Active_HeartRate_MonitorReading_EventHandlerUpdateAssignCollaborator",
+    "target" : "procedure/patient.monitoring.HeartRate.Active_HeartRate_MonitorReading_EventHandlerGet"
+  }, {
+    "source" : "procedure/patient.monitoring.HeartRate.Active_HeartRate_MonitorReading_EventHandlerUpdateAssignCollaborator",
+    "target" : "services/patient.monitoring.HeartRate"
+  }, {
+    "source" : "app/patient.monitoring.HeartRate_MonitorReading_EventHandler",
+    "target" : "procedure/patient.monitoring.HeartRate.Active_HeartRate_MonitorReading_EventHandlerUpdateAssignCollaborator"
+  } ],
+  "name" : "PatientMonitor",
+  "options" : {
+    "description" : "Advanced Collaborations\n\nThis project contains the patient.monitoringHeartRate Service which defines the MonitorReading Event Handler. This event Handler detects when a patient has an unusal and dangerous heart rate. First, the patient is asked whether they are okay or require assistance. If the patient replies that they require assistance or does not reply within 90 seconds, the an array of nearby first responders is computed. The closest responder is notified. If the responder does not reply within 1 minute, the notification is retracted and the next nearest first responder is notified. When a first responder replies, they are tracked until they reach the patient's location and the collaboration completes. If no first responder reply to the notification, the collaboration fails.\n",
```

typos: the an   & reply & unusal
If Handler is capitalized then capitalize both: Event Handler 
Also add a note about updating username, if you add 4 rows of data for the FirstResponder type.
```suggestion
    "description" : "Advanced Collaborations\n\nThis project contains the patient.monitoringHeartRate Service, which defines the MonitorReading Event Handler. This Event Handler detects when a patient has an unusual and dangerous heart rate. First, the patient is asked whether they are okay or require assistance. If the patient replies that they require assistance or does not reply within 90 seconds, then an array of nearby first responders is computed. The closest responder is notified. If the responder does not reply within 1 minute, the notification is retracted and the next nearest first responder is notified. When a first responder replies, they are tracked until they reach the patient's location and the collaboration completes. If no first responder replies to the notification, the collaboration fails.\n",
```